addons/utm/models/utm_mixin.py

```diff
@@ -58,10 +59,10 @@ def tracking_fields(self):
             ('utm_medium', 'medium_id', 'odoo_utm_medium'),
         ]
 
-    def _find_or_create_record(self, model_name, name):
+    def _find_or_create_record(self, model_name, name, lowercase=False):
         """Based on the model name and on the name of the record, retrieve the corresponding record or create it."""
         Model = self.env[model_name]
-
+        name = name.lower() if lowercase else name
```

Comment:
This won't work for existing records that weren't lowered right?
Shouldn't you lower in the caller instead?

Reply:
but ilike would also find uppercase records, like: FACEBOOK. and it might attach to that. i don't know, i will do it if u insist